### PR TITLE
Fix issue with composer dependencies & FOSRest config

### LINKED ISSUE #2421: Wrong MIME Type returned for CSS files (text/html instead of text/css)
Hi,

After updating Sylius to the last version (0.13 taken from github friday 30/09) in my project, i'm facing a non loading CSS file.
Capture of firefox console log :
bug-sylius

After some investigation I found the problem (thanks to this [post] (https://stackoverflow.com/questions/22331505/custom-content-type-in-symfony2-assetic-phpless) ).

It was a recent modification made by @loevstroem in the main.yml config file (src/Sylius/Bundle/CoreBundle/Resources/config/app/main.yml).

The `stop: true` directive was removed from the `- { path: '^/', stop: true }` line and wass causing this problem.

I tried to put `stop: true` again and everything is working fine, CSS are loading correctly.

I'm using less in my project (leafo/lessphp), and maybe this could be an important thing.

In the stackoverflow post, venimus (last post) said to add a rule to prevent exclude assetic, maybe this could solve the problem ?

Any help is appreciated ;)
Thx

### LINKED ISSUE #2425: Cannot find OmnipayBridge payment
## Master branch version 0d015923436bfc9e540aaaaaaaed08d86ff6430a

While installing vendors it can not find OmnipayBridge class and gives error during cache cleaning. 

solution is to add "payum/omnipay-bridge": "0.13.*" to composer.json

### PR DESCRIPTION
**Pull request by stloyd**
Closes #2425.
Fixes #2421.

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
Travis... Where are you?

**Comment by loevstroem** [comment]
Thanks Joseph (y)

**Comment by psyray** [comment]
Thx

Another strange issue when I do a vendor update `composer.phar update --prefer-dist --profile -vvv`, it upgrades me to symfony 2.6.
I have to remove the vendors dir and composer.lock file, run `composer.phar install --prefer-dist --profile -vvv` to have latest symfony 2.3 version.

Is this normal ?

File: composer.json [hunk 1]
```diff
@@ -75,7 +75,7 @@
         "willdurand/hateoas-bundle":            "@stable",
         "winzou/state-machine":                 "~0.1",
         "winzou/state-machine-bundle":          "~0.2",
-        "payum/omnipay-bridge": "0.13.*"
+        "payum/omnipay-bridge":                 "~0.12"
```
Comment (makasim) [inline composer.json]: @stloyd  0.12.*

Comment (stloyd) [inline composer.json]: It's covered by `payum` version, composer locks himself on `0.12` so there is no need to change.

Comment (stloyd) [inline composer.json]: Also there should be `provide` in `payum/payum` package as there is `replace` note (for version `0.12` ofc).

**Comment by psyray** [comment]
Always same issue with symfony updating to 2.6 while doing an update.
Workaround is delete vendor dir, composer.lock and do an install.

Is this problem happens to someone ?

**Pull request merged**